templates/deployments/appname/app.jsonnet.tpl

```diff
@@ -169,6 +169,7 @@ local all = {
 				Path: '/run/secrets/outreach.io/launchdarkly/sdk-key',
 			},
 			flagsToAdd: {
+        appName: app.name,
```

Comment:
suggestion: according to the snapshot test failure, this should be tabs to be consistent with the other jsonnet around it.
```suggestion
				appName: app.name,
```

Reply:
But also the tests need to be regenerated (Run `make test` after this change is in, and commit the updated snapshots).

Reply:
Thank you, updated (sorry I'm not a golang dev so I'm learning as I go)